### DISCUSSION TITLE
Welcome to PaperWM Discussions!

### DISCUSSION THREAD

**Discussion by jtaala** [paperwm/PaperWM#708] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

**Comment by jtaala** [comment]
Note that we're soon intending to deprecate / stop our use of zulip (see https://github.com/paperwm/PaperWM/issues/688#issuecomment-1811332084) and use discussions here instead.

**Comment by basings** [comment]
When I moved to PaperWM I was looking for a matrix space since a lot of projects nowadays have one. It's very easy to connect and especially good for users who aren't on github every day. It's also more ephermal than github discussions and you have another path for communicating things.

**Reply by Thesola10** [reply]
is being ephemeral a good thing? I like perusing through existing issues and discussions as reference for the state of things or why certain things were done a specific way. it's a form of living documentation imo.
